### DISCUSSION TITLE
ip Reconciler

### DISCUSSION THREAD

**Discussion by Raouf-ai** [k8snetworkplumbingwg/whereabouts#307] [General]
Since the IP-Reconcile code has been moved to IP Control Loop in the latest Image i have few questions related to the same. 

1. Earlier we used to run the reconcile job with command `/ip-reconciler ` however that is now not working with the latest image for obvious reasons and it couldn't locate the directory. In order to run this which command should we mention in the CronJob of K8s now; Or is it simply not needed anymore?

2.Or should we replace `/ip-reconciler `with `ip-controlloop`

**Comment by infinitydon** [comment]
@dougbtv - With regards to the above, can you kindly clarify the current state of re-conciliation. As per below:

```
[Whereabouts 0.5.2](https://github.com/k8snetworkplumbingwg/whereabouts/releases/tag/v0.5.2)
This is the last release that will use only the cronjob style reconciler. In upcoming releases we will switch to a controller for reconciliation.
```
You mentioned that this will be switched to a controller.

Also the master documentation is still referring to the ip-reconciler cronjob.
